src/main/java/com/venom/CRUD/model/Employee.java

```diff
@@ -0,0 +1,59 @@
+package com.venom.CRUD.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "Employee")
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "surname")
+    private String surname;
+    @Column(name = "mob")
+    private long mob;
+
+    public Employee() {
+    }
```

Comment:
can use lombok annotations for generating getter and setter methods like `@Data`